### ISSUE TITLE
Importing into more than one locale...

### ISSUE THREAD

**Issue by etheya** [boboldehampsink/import#83]
Ive been trying but with little success so far to allow the import to populate 2 locales... So the first been english and the 2nd variation of the entry been french... 

So for example they have a different TITLE or a given FIELD...

Is it possible to do this?

**Comment by boboldehampsink** [comment]
Try the develop branch, it's in there

**Comment by etheya** [comment]
Oh!... will have a look.... your a star if it is :)

**Comment by etheya** [comment]
Can i use the develop branch ok?

**Comment by boboldehampsink** [comment]
Yeah, its as good as stable. Will release it very soon.

**Comment by etheya** [comment]
Your a star i will try it out...

**Comment by etheya** [comment]
Cant see anything different... unless I'm missing something?

**Comment by boboldehampsink** [comment]
You can now use a column "Locale" to specify what locale that row should be imported into

**Comment by etheya** [comment]
Ah.. so if i have a column called Locale... i can specify information in that row to be imported... if its has say FR...

**Comment by boboldehampsink** [comment]
Yeah, you should say "fr" in that case then.

**Comment by boboldehampsink** [comment]
This was just released on master, https://github.com/boboldehampsink/import/releases/tag/0.8.27

**Comment by etheya** [comment]
Ok will try that out. Thanks

**Comment by etheya** [comment]
Not sure if I'm going mad here... I've imported the first set of entries in ENGLISH... and thats gone fine. I now want to import the FRENCH (title) and a few other fields... I've created a new column called LOCALE.. and have specified fr in this.. when i run the import tho how do i make sure its adding new data to the existing entry. I can't choose the title as i want to add the french title...

**Comment by boboldehampsink** [comment]
Use the replace option

**Comment by etheya** [comment]
But which do you use to check for the right entry? If I use the title (en) I then can't choose the title (fr) as its not available.

**Comment by boboldehampsink** [comment]
You use a combination of title and locale

**Comment by etheya** [comment]
Still bit confused.. 

So if I have 3 columns in a csv file. 

Title (en)
Title (fr)
Locale 

First time round I run the import with append.. I choose not to import locale, the I map the Title field to Title(en). 

This imports everything fine. 

Then... 

I want to add the French alternative entry titles. So.. 

I import using replace.. Map locale to the locale  column (which is set to fr) but then if I map the Title to the Title (fr) column.. How does it know which entry is which without using some criteria checkbox?

That's where I'm confused as I thought I had to map something so it updates the right entry?

Thanks sorry for not quite getting it

**Comment by boboldehampsink** [comment]
Mmm, maybe you can find your answer here https://github.com/boboldehampsink/import/issues/62

**Comment by etheya** [comment]
Thanks Bob… will have a look here and try..

Kind Regards

Brian Coult
Mobile Consultant | Designer | ExpressionEngine Developer
Etheya Digital Creativity Ltd

t. 07463 240076
e. brian@etheya-digitalcreativity.com <mailto:brian@etheya-digitalcreativity.com>
w. www.etheya-digitalcreativity.com <http://www.etheya-digitalcreativity.com/>

Skype: etheya-bc
LinkedIn: http://www.linkedin.com/pub/brian-coult/3/902/127 <http://www.linkedin.com/pub/brian-coult/3/902/127>

**Comment by etheya** [comment]
Got this working.... but only after i pulled down using EXPORT another field that i could utilise to check criteria...

If you have only the title changing then its not really possible to do this as you need to use the LOCALE and the TITLE (en) to be checked for criteria.... which means you then can't update the TITLE (fr)... it would be fine if you were changing another field in the entry, but not for the title itself. So i pulled down the slug/ID once i ran the initial import... added that to the csv file, and the used the ID and LOCALE as the criteria and it imported nicely.

Might be something to think about as a requirement tho... 

Thanks again... nice add-on.

**Comment by etheya** [comment]
Got this sorted... but have run into another issue, but will create a new post for that.. Consider this closed...
